spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe "ingredients index page", type: :feature do
+    
+    # User Story 1
+    it "ingredients index page lists name and cost" do
```

Comment:
Nice job with these feature tests! As the stories get more complex, you may want to get used to copying/pasting the story in and just breaking it down line by line as well as summarizing it in the `it` block as you do here, just to be sure no details are missed. But this does look great!